examples/shaders/resources/shaders/glsl330/base_lighting_instanced.vs

```diff
@@ -0,0 +1,36 @@
+#version 330
```

Comment:
Not sure if this file should be here, it would require the instancing example.

Reply:
Added an example. Maybe not necessary for something low level, but there it is :)